ipaclient/install/ipa_epn.py

```diff
@@ -238,12 +238,35 @@ def add_options(cls, parser):
 
     def validate_options(self):
         super(EPN, self).validate_options(needs_root=True)
-        if self.options.to_nbdays:
+        if self.options.to_nbdays is not None:
+            try:
+                if int(self.options.to_nbdays) < 0:
+                    raise RuntimeError('Input is negative.')
+            except Exception as e:
+                self.option_parser.error(
+                    "--to-nbdays must be a positive integer. "
+                    "{error}".format(error=e)
+                )
             self.options.dry_run = True
-        if self.options.from_nbdays and not self.options.to_nbdays:
+        if self.options.from_nbdays is not None:
+            try:
+                if int(self.options.from_nbdays) < 0:
+                    raise RuntimeError('Input is negative.')
+            except Exception as e:
+                self.option_parser.error(
+                    "--from-nbdays must be a positive integer. "
+                    "{error}".format(error=e)
+                )
+        if self.options.from_nbdays is not None and not self.options.to_nbdays:
```

Comment:
Shouldn't it be also `...and self.options.to_nbdays is not None` ? otherwise the `--from-nbdays 10 --to-nbdays 0` arguments would trigger this message.

Reply:
Good catch too, thanks, fixed.